src/explore-education-statistics-common/src/components/ContentSectionIndex.tsx

```diff
@@ -171,10 +180,20 @@ const ContentSectionIndex = ({
           <h3 className="govuk-heading-s">In this section</h3>
 
           <ul className="govuk-body-s">
-            {elements.map((element, index) => (
-              // eslint-disable-next-line react/no-array-index-key
-              <li key={index}>
-                <a href={`#${element.id}`}>{element.textContent}</a>
+            {headingsList.map(listItem => (
+              <li key={listItem.id}>
+                <a href={`#${listItem.id}`}>{listItem.textContent}</a>
+                {listItem.children.length > 0 && (
```

Comment:
`listItem.children.length > 0` can just be `listItem.children.length` as we generally write it elsewhere

Reply:
When it's in the jsx like this you need the `> 0` as it outputs 0 on the frontend otherwise.

Reply:
Ahh good to know :)  Ta!